content/cucumber/configuration.md

```diff
@@ -1,13 +1,75 @@
 ---
 title: Cucumber Configuration
-subtitle: cucumber.yml, environment variables
+subtitle: cucumber.yml, type registry
 polyglot: true
 ---
 
+
+# Type Registry Configuration
+
+{{% block "java" %}}
+The type registry is used to configure parameter and data table types. It can be configured by placing an implementation 
```

Comment:
"parameter and data table types" is a little confusing to me; do you mean "parameters" or "parameter types". Please specify as otherwise it makes the reader wonder.

Reply:
Eh. I guess 'parameter- and data table types'? Not sure how that construct works in English actually.